third_party/terraform/resources/resource_compute_router_nat.go

```diff
@@ -302,8 +279,105 @@ func resourceComputeRouterNatRead(d *schema.ResourceData, meta interface{}) erro
 	return nil
 }
 
-func resourceComputeRouterNatDelete(d *schema.ResourceData, meta interface{}) error {
+func resourceComputeRouterNatUpdate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
 
```

Comment:
I like how the diff tool decided the empty line was the important thing to preserve here vs the function header lol

Reply:
🤷‍♀